src/components/Header/index.jsx

```diff
@@ -0,0 +1,11 @@
+const Header = () => {
+    return (
+        <header>
+            <div className={'row align-items-center justify-content-between'}>
+                <h2 className={'col-6 my-3'}>Most recent jobs</h2>
```

Comment:
Probably this will need to be passed in as a prop eventually, I would change it to do this now

Reply:
Changed this so that headerText is state default set to Most recent jobs and passed that in.  Is that along the right lines?